test/integration/replicasets/utils.py

```diff
@@ -39,7 +39,10 @@ def __init__(self, cli, project, instances_list, replicasets_list=[], vshard_gro
                 }
             })
 
-        write_conf(os.path.join(project.path, DEFAULT_CFG), instances_conf)
+        instances_conf_path = os.path.join(project.path, DEFAULT_CFG)
+        if not os.path.exists(instances_conf_path):
+            write_conf(os.path.join(project.path, DEFAULT_CFG), instances_conf)
```

Comment:
```suggestion
            write_conf(instances_conf_path, instances_conf)
```

Reply:
Sorry, I've commited this change without applying your suggestion.

Reply:
I don't resent you in the least